### PR TITLE
[ACS-4636] create tag count param support

### PR DESCRIPTION
**Pull request by pksingh41**
https://alfresco.atlassian.net/browse/ACS-4636

### CODE REVIEW

File: remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java [hunk 1]
```diff
@@ -453,7 +453,7 @@ public void testAddTags()
         List<Tag> tags = tagNames.stream().map(name -> Tag.builder().tag(name).create()).collect(toList());
         given(taggingServiceMock.addTags(CONTENT_NODE_REF, tagNames)).willReturn(pairs);
```
Comment (kavitshah-gl) [inline remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java:455]: given(parametersMock.getInclude()).willReturn(List.of("count")); should be added to initialize the mock value for count.

**Review by kavitshah-gl — CHANGES_REQUESTED** [review]
Looks fine with small change required in testing class.

File: remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java (hunk 1 above)
Comment (pksingh41) [reply remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java:459]: Done

**Review by kavitshah-gl — APPROVED** [review]
Seems to be okay.

**Pull request merged**